code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -267,6 +267,18 @@
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 100, 1)
 
 
+/obj/machinery/atmospherics/components/unary/vent_scrubber/high_volume
+	name = "large scrubber"
+	idle_power_usage = 25
+	active_power_usage = 150
+	volume_rate = 1000
```

Comment:
5x the volume rate at less than 3x the active power usage? I'm iffy on the idea of a large scrubber in the first place, but if it *is* included, I think it should have much higher power usage to compensate for the added utility.

Reply:
Values are now 6x for the large scrubber